Today Extension/NoRoutesTableViewCell.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NoRoutesTableViewCell.swift
+//  Today Extension
+//
+//  Created by Yana Sang on 2/28/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class NoRoutesCell: UITableViewCell {
```

Comment:
Nitpick: `NoRoutesAvailableTableViewCell` or `EmptyStateTableViewCell` is a bit better name to communicate what the cell does